code/game/objects/effects/decals/decal.dm

```diff
@@ -0,0 +1,15 @@
+/obj/effect/turf_decal
+	icon = 'icons/turf/decals.dmi'
+	icon_state = "warningline"
+	layer = TURF_DECAL_LAYER
+
+/obj/effect/turf_decal/Initialize()
+	..()
+	qdel(src) // return INITIALIZE_HINT_QDEL <-- Doesn't work
```

Comment:
Our INITIALIZE_HINT_QDEL does not work. It simply doesn't seem to delete the object.

Reply:
I suspect something is broken with how we `Initialize` things.

I'm working on an unrelated PR that adds a timer after `New` or `Initialize`, and the timer will never start.

Reply:
Did you ever manage to pin down this issue? Or is it still flaily?

Reply:
I haven't been able to, no.